tests/test_api.py

```diff
@@ -297,26 +349,32 @@ def test_metadata_timestamp(self):
 
         hashes = {'sha256': '0ae9664468150a9aa1e7f11feecb32341658eb84292851367fea2da88e8a58dc'}
         fileinfo = copy.deepcopy(timestamp.signed.meta['snapshot.json'])
-        fileinfo['hashes'] = hashes
-        fileinfo['version'] = 2
-        fileinfo['length'] = 520
-
-        self.assertNotEqual(timestamp.signed.meta['snapshot.json'], fileinfo)
-        timestamp.signed.update(2, 520, hashes)
-        self.assertEqual(timestamp.signed.meta['snapshot.json'], fileinfo)
+        fileinfo.hashes = hashes
+        fileinfo.version = 2
+        fileinfo.length = 520
```

Comment:
nit, This would be much more understandable as `fileinfo = MetaInfo(...)` as well